tests/test_user_add.py

```diff
@@ -41,7 +43,7 @@ def test_add_user_with_researcher(client):
         data=json.dumps(first_new_user),
         content_type="application/json",
     )
-    assert response.status == "403 FORBIDDEN"
+    response.status_code == http.HTTPStatus.FORBIDDEN
```

Comment:
I think it's nice to keep the `assert` before the comparison?

Reply:
👍